artemis-jms-client/src/main/java/org/apache/activemq/artemis/jms/client/ActiveMQSession.java

```diff
@@ -634,8 +634,8 @@ public MessageConsumer createSharedConsumer(Topic topic, String name, String mes
       }
       checkTopic(topic);
       ActiveMQTopic localTopic;
-      if (topic instanceof ActiveMQTopic) {
-         localTopic = (ActiveMQTopic) topic;
+      if (topic instanceof ActiveMQTopic qTopic) {
```

Comment:
qException was strange, but qTopic is about the most nonsense thing it could have picked :)

Here and all the others below

Reply:
`qTopic`...lol